doc/source/guide/acquiring_data/fido.rst

```diff
@@ -0,0 +1,75 @@
+---------------------------------------
+Finding and Downloading Data using Fido
+---------------------------------------
+
+This guide outlines how to search for and download data using SunPy's
+Federated Internet Data Obtainer...or more usually (and
+sanely) referred to as Fido.  Fido is a unified interface for seeking
+and downloading solar physics data irrespective of the underlining
+client or webservice through which the data is obtained, e.g. VSO,
+JSOC, etc.  It therefore supplies a single, easy and consistent way to
+obtain most forms of solar physics data.
+
+Import
+------
+
+SunPy's Fido module is in ``sunpy.net``.  It can be imported as follows::
+
+    >>> from sunpy.net import Fido, attrs as a
+
+Searching for Data Using Fido
+-----------------------------
+
+To search for data with Fido, your query needs at minimum a start time,
```

Comment:
This is not strictly true. JSOC does not require an instrument, and in fact after #2188 is merged it wont even require a time.

(I believe the requirement for a Time to be present is currently coded into Fido, I guess that will need to be relaxed.)